helpers/helpers.rb

```diff
@@ -1,3 +1,131 @@
+# frozen_string_literal: true
+
+def search(uuid)
+  visit STAFF_URL
+
+  fill_in 'global-search-box', with: uuid
+  find('#global-search-button').click
+end
+
+def login_admin
+  visit STAFF_URL
+
+  fill_in 'username', with: 'admin'
+  fill_in 'password', with: 'admin'
+
+  click_on 'Sign In'
+
+  expect(page).to have_content 'Welcome to ArchivesSpace'
+  expect(page).to have_content 'Your friendly archives management tool.'
+  element = find('.global-header .user-container')
+  expect(element.text.strip).to eq 'admin'
+
+  # Ensure the system has at least one repository
+  begin
+    element = find('.alert.alert-info.with-hide-alert')
+
+    if element.text == 'To create your first Repository, click the System menu above and then Manage Repositories.'
+      click_on 'System'
+      click_on 'Manage Repositories'
+      click_on 'Create Repository'
+
+      fill_in 'repository_repository__repo_code_', with: 'repository_test'
+      fill_in 'repository_repository__name_', with: 'Repository Test'
+      find('#repository_repository__publish_').check
+      click_on 'Save'
+
+      expect(find('.alert.alert-success.with-hide-alert').text).to eq 'Repository Created'
+      expect(find('.alert.alert-info.with-hide-alert').text).to eq 'Repository is Currently Selected'
```

Comment:
I think this should be a separate function, since it's introducing a side effect of logging in as admin.  Otherwise consider renaming the function to something like 'login_admin_and_create_repository'

Reply:
Yes you are right. I have extracted the responsibility to create a test repository, to separate function.